python/lsst/ts/standardscripts/maintel/m1m3/check_actuators.py

```diff
@@ -77,25 +80,29 @@ def __init__(self, index, add_remotes: bool = True):
 
     async def assert_feasibility(self):
         """Verify that the system is in a feasible state before
-        running bump test.
+        running bump test. Note that M1M3 mirror should be in lowered
+        position.
         """
 
         for comp in self.mtcs.components_attr:
             if comp != "mtm1m3":
                 self.log.debug(f"Ignoring component {comp}.")
                 setattr(self.mtcs.check, comp, False)
 
-        # Check all enabled, liveliness and the m1m3 detailed state
+        # Check all enabled and liveliness
         await asyncio.gather(
             self.mtcs.assert_all_enabled(),
             self.mtcs.assert_liveliness(),
-            self.mtcs.assert_m1m3_detailed_state(
-                {
-                    DetailedState.PARKED,
-                    DetailedState.PARKEDENGINEERING,
-                }
-            ),
         )
+        # Check if m1m3 detailed state is either PARKED or PARKEDENGINEERING
+        expected_states = {DetailedState.PARKED, DetailedState.PARKEDENGINEERING}
+        try:
+            await self.mtcs.assert_m1m3_detailed_state(expected_states)
+        except AssertionError:
+            raise RuntimeError(
+                f"M1M3 detailed should be in one of the following detailed states: {expected_states}."
```

Comment:
The message shall be a variation of:
```
M1M3 needs to be parked for the bump test to proceed. Please lower the mirror, or enable M1M3 CSC.
```

Reply:
I was inclined to keep the message here just to give a more friendly message like "lower the mirror", but not sure if I got @pkubanek's comments. If the mirror is raised, will enabling the M1M3 CSC put it in a parked state? Can you clarify this, please?

The message would be then, something like:

"_Please park M1M3 before proceeding with the bump test. This can be done by lowering the mirror or enabling the M1M3 CSC._"